frontend/src/components/Header.tsx

```diff
@@ -1,11 +1,10 @@
 "use client";
-
+import GrantsLogo from "public/img/grants-logo.png";
 import clsx from "clsx";
 import { useFeatureFlags } from "src/hooks/useFeatureFlags";
-import { assetPath } from "src/utils/assetPath";
-
 import { useTranslations } from "next-intl";
 import Link from "next/link";
+import Image from "next/image";
```

Comment:
Updated this to use the same image strategy as the `Footer` component because this one doesn't work in React 19/Next 15 (not sure specifically why).